hanami-model.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "rom",             "~> 5.1"
   spec.add_runtime_dependency "rom-repository",  "~> 5.1"
   spec.add_runtime_dependency "rom-sql",         "~> 3.0"
-  spec.add_runtime_dependency "dry-types",       "~> 1.1"
+  spec.add_runtime_dependency "dry-types",       "~> 1.2"
```

Comment:
WDYT if we set `1.3` version instead `1.2`? in this case, we'll able to use latest dry-system and I'll  have an ability to test my library for dry-system dependency graph 🤔

Reply:
IIRC it wasn't available at the time I made the changes. I'll give it a try and see how it goes.

Reply:
Okay, updating to 1.3 introduces new issues. I'll merge this, and open a new one to 1.3 otherwise it will get confusing.